packages/aragon-wrapper/src/identity/AddressBookIdentityProvider.js

```diff
@@ -0,0 +1,64 @@
+import AddressIdentityProvider from './AddressIdentityProvider'
+import { first, map } from 'rxjs/operators'
+import { getCacheKey } from '../utils/index'
+
+const addressBookAppIds = [
+  '0x32ec8cc9f3136797e0ae30e7bf3740905b0417b81ff6d4a74f6100f9037425de'
+  // TODO Add in App Ids for rinkeby and mainnet appIds
```

Comment:
I think Aragon eventually wants to develop a way to detect third-party ID providers that don't need to be merged into their repo, so this would improve that process significantly

Reply:
Should we add a TODO for now, noting that this should eventually become a permissionless system?

Reply:
This is the kind of thing that won't get developed until there's demand for it, either internally or externally due to this maintenance process becoming unsustainable. I'll update the note and if Brett wants it out, we'll remove it